backend/app/Http/Requests/Web/Admin/GeneratePairingCodeRequest.php

```
@@ -0,0 +1,45 @@
+            'name' => ['required', 'string', 'max:100'],
+            'line_id' => [
+                'nullable', 'integer',
+                Rule::exists('lines', 'id')->where(fn ($q) => $q->where('tenant_id', $tenantId)),
+            ],
+            // Workstation has no tenant_id of its own — it is scoped through its
+            // (tenant-scoped) Line, so validate via whereHas('line'), which the
+            // Line global scope narrows to the current tenant.
+            'workstation_id' => [
+                'nullable', 'integer',
+                function ($attribute, $value, $fail) {
+                    if ($value !== null && ! Workstation::whereKey($value)->whereHas('line')->exists()) {
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Workstation::whereKey().


(staticMethod.notFound)